packages/salesforcedx-vscode-core/package.json

```diff
@@ -150,14 +150,14 @@
         {
           "id": "metadata",
           "name": "Metadata",
-          "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking"
+          "when": "sfdx:project_opened && (sfdx:default_username_has_no_change_tracking || !sfdx:is_scratch_org)"
```

Comment:
Had to add this context variable in order to keep the Deploy* and Retrieve* commands exposed for source-tracked Sandboxes but not yet expose them for Scratch Orgs.

Reply:
Do the deploy/retrieve commands not yet work for scratch orgs?

Reply:
They work to deploy and retrieve the components, but they do not fix up source tracking correctly, since they don't have the source tracking library in the flow.  

I broke the original PR up to split out the work in this PR, and the next two PRs will be to "fix Retrieve for source-tracked sandboxes" (to add in STL #4773), and then  "fix Deploy for source-tracked sandboxes" (to add in STL).  

After those two, we will expose deploy/retrieve for Scratch orgs (now that they will be handling source tracking correctly).  And finally, expose Org Browser.